django/db/models/expressions.py

```diff
@@ -620,6 +624,8 @@ def as_sql(self, compiler, connection):
 
 
 class RawSQL(Expression):
+    contributes_to_group_by = True
```

Comment:
Yes. The last commit was related to my previous comment—I don’t think there cases RawSQL can be used as an ORDER BY expression. An ORDER BY expression can only be generated through (as far as I can find) `Meta.ordering`, `.order_by` (basically the same thing), and `.extra({'order_by': ...})`. In all those cases, the expression passed in will be wrapped in an `OrderBy` expression, so there is no use to mark `RawSQL` with `contributes_to_group_by` because it is inspected directly in `Compiler.get_group_by `, but only within as an `OrderBy(RawSQL(...))`. In order to make the property meaningful, it should respect the value provided by contained expressions; an expression should be included if any of the nested expressions contain a column reference, or is a `RawSQL`.

I’m going to try out some cases using `RawSQL` as an ORDER BY expression. Honestly I don’t know how the SQL is generated, so I’ll report back after I have a better idea how it works.